lib/gsheets/gsheets.js

```diff
@@ -131,8 +140,9 @@ const formatValues = values => values.slice(3).join('\t');
  * @return {Promise}
  */
 function appendResults(auth, valuesToAppend, options) {
-  var sheets = google.sheets('v4');
-  const values = [valuesToAppend];
+  const sheets = google.sheets('v4');
+  // clone values to append
+  const values = valuesToAppend.slice();
```

Comment:
I'd rather do a JSONparse(jsonstringify(valuestoappend)) or Object.assign([], valuesToAppend) to clone instead.